rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Would that expression work with the current way we are filtering hosts here?
https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/rpc_maas/tasks/ensure_local_checks.yml#L27